app/actions/bonds.ts

```diff
@@ -1,104 +1,102 @@
-import { ContractInterface, ethers, providers } from "ethers";
+import { ethers, providers } from "ethers";
 import { Thunk } from "state";
 import { setBond } from "state/bonds";
 import { OnStatusHandler } from "./utils";
 import { setBondAllowance } from "state/user";
-import { formatUnits, getJsonRpcProvider } from "@klimadao/lib/utils";
+import {
+  formatUnits,
+  getJsonRpcProvider,
+  getContract,
+} from "@klimadao/lib/utils";
 import { addresses, Bond } from "@klimadao/lib/constants";
-import Depository from "@klimadao/lib/abi/KlimaBondDepository_Regular.json";
 import PairContract from "@klimadao/lib/abi/PairContract.json";
 import BondCalcContract from "@klimadao/lib/abi/BondCalcContract.json";
-import OhmDai from "@klimadao/lib/abi/OhmDai.json";
-import IERC20 from "@klimadao/lib/abi/IERC20.json";
-import KlimaProV2 from "@klimadao/lib/abi/KlimaProV2.json";
 
-const getBondAddress = (params: { bond: Bond }): string => {
-  return {
-    klima_mco2_lp: addresses["mainnet"].bond_klimaMco2Lp,
-    klima_bct_lp: addresses["mainnet"].bond_klimaBctLp,
-    klima_usdc_lp: addresses["mainnet"].bond_klimaUsdcLp,
-    bct: addresses["mainnet"].bond_bct,
-    bct_usdc_lp: addresses["mainnet"].bond_bctUsdcLp,
-    mco2: addresses["mainnet"].bond_mco2,
-    nbo: addresses["mainnet"].bond_nbo,
-    ubo: addresses["mainnet"].bond_ubo,
-    inverse_usdc: addresses["mainnet"].klimaProV2,
-  }[params.bond];
-};
+// All Bonds without Inverse
```

Comment:
why can't we do
`inverse_usdc: "klima"` in here?

Thinking out loud,
I guess I made it confusing by calling it `inverse_usdc`. All the other bonds are named after the "input" token:
input bctUsdcLP -> get KLIMA

but in this case its:
input KLIMA -> get USDC

the challenge is we could theoretically in the future offer `input KLIMA -> get BCT` as an inverse.

Reply:
Added `inverse_usdc: "klima"`. to the mapping